src/lib/api.js

```diff
@@ -0,0 +1,114 @@
+import axios from 'axios';
+import pickBy from 'lodash/pickBy';
```

Comment:
this is 4.5kb minfied and gzipped, https://bundlephobia.com/result?p=lodash.pickby@4.6.0, is it a big deal to avoid it?

Reply:
Done.